Sources/Commands/SwiftTool.swift

```diff
@@ -379,6 +379,15 @@ public class SwiftTool<Options: ToolOptions> {
         binder.bind(
             option: parser.add(option: "--destination", kind: PathArgument.self),
             to: { $0.customCompileDestination = $1.path })
+        binder.bind(
+            option: parser.add(option: "--triple", kind: String.self),
+            to: { $0.customCompileTriple = try Triple($1) })
+        binder.bind(
+            option: parser.add(option: "--sdk", kind: PathArgument.self),
+            to: { $0.customCompileSDK = $1.path })
+        binder.bind(
+            option: parser.add(option: "--toolchain-bin-dir", kind: PathArgument.self),
```

Comment:
I think it might be better to call this `--toolchain` and expect the toolchain directory. SwiftPM can then append `usr/bin` when assigning the value in destination.

Reply:
Done.